tests/smoke/events.js

```diff
@@ -155,6 +156,44 @@
 					observer.assertRootIs(inner);
 					observer.assert(["spellCheckAbort"]);
 				}
+		},
+		'test spellcheck can batch ajax calls for a document with multiple blocks': function() {
+			var bot = this.editorBot,
+				editor = bot.editor,
+				resumeAfter = bender.tools.resumeAfter,
+				observer = observeSpellCheckEvents(editor),
+				fiveBlockHtml = '<p>This</p><p>is</p><ul><li>five</li><li>block</li></ul><p>test</p>',
+				sixBlockHtml = '<p>This</p><p>is</p><p>a</p><ul><li>six</li><li>block</li></ul><p>one</p>';
```

Comment:
I guess it's not 100% necessary.  I just figured it would be useful to confirm that by increasing by one more block over the limit, we actually triggered a second request.